src/mappings/event/loans.ts

```diff
@@ -545,46 +545,41 @@ export async function liquidateLoan(
     entityBuffer: EntityBuffer
 ): Promise<void> {
     const rawEvent = new LoansLiquidatedBorrowEvent(ctx, item.event);
-    // {liquidator: Uint8Array,
-    // borrower: Uint8Array,
-    // liquidationCurrencyId: v1021000.CurrencyId,
-    // collateralCurrencyId: v1021000.CurrencyId,
-    // repayAmount: bigint,
-    // collateralUnderlyingAmount: bigint}
-    // let accountId: Uint8Array;
-    // let myCurrencyId: CurrencyId_V1020000|CurrencyId_V1021000;
-    // let amount: bigint;
-    // let e;
-    // if (rawEvent.isV1020000) {
-    //     e = rawEvent.asV1020000;
-    //     [ accountId, myCurrencyId, amount ] = e;
-    // }
-    // else if (rawEvent.isV1021000) {
-    //     e = rawEvent.asV1021000;
-    //     accountId = e.accountId;
-    //     myCurrencyId = e.currencyId;
-    //     amount = e.amount;
-    // }
-    // else {
-    //     ctx.log.warn(`UNKOWN EVENT VERSION: LoansRepaidBorrowEvent`);
-    //     return;
-    // }
-    // const currency = currencyId.encode(myCurrencyId);
-    // const height = await blockToHeight(ctx, block.height, "Redeemed");
-    // const account = address.interlay.encode(accountId);
-    // const comment = `${getFirstAndLastFour(account)} withdrew ${await friendlyAmount(currency, Number(amount))} from deposit`;
-    // await entityBuffer.pushEntity(
-    //     Deposit.name,
-    //     new Deposit({
-    //         id: item.event.id,
-    //         height: height,
-    //         timestamp: new Date(block.timestamp),
-    //         userParachainAddress: account,
-    //         token: currency,
-    //         amountWithdrawn: amount,
-    //         comment: comment// expand to 3 tokens: qToken, Token, equivalent in USD(T)
-    //     })
-    // );
+    let amountRepaid: bigint;
+    let amountRepaidToken;
+    let seizedCollateral: bigint;
+    let seizedCollateralToken;
+    let liquidationCost: bigint;
+    let liquidationCostToken;
+    let e;
+    if (rawEvent.isV1021000) {
+        e = rawEvent.asV1021000;
+        amountRepaid = e.repayAmount;
+        amountRepaidToken = currencyId.encode(e.liquidationCurrencyId);
+        seizedCollateral = e.collateralUnderlyingAmount;
+        seizedCollateralToken = currencyId.encode(e.collateralCurrencyId);
+    }
+    else {
+        ctx.log.warn(`UNKOWN EVENT VERSION: LoansLiquidatedBorrowEvent`);
+        return;
+    }
+    
+    liquidationCost = seizedCollateral - amountRepaid;
```

Comment:
Do we expect seizedCollateral and amount Repaid to be the same token? (once again this would be easy to know with a test, but I have had issues with the kinsugi testnet)

Reply:
I suspect they're not the same, but that's just me guessing.

Pinging @peterslany and @daniel-savu, they'll be able to confirm/deny.

Reply:
I think we can compute `liquidationCost` in the squid!